src/Mapbender/CoreBundle/Resources/public/mapbender.element.redlining.js

```diff
@@ -37,6 +37,15 @@
             }
             this._trigger('ready');
             this._ready();
+
+            //put rl layer on top of layer stack if a source is added, i.e. by wms loader
+            var self = this;
```

Comment:
Would be better to use .bind(this); 
```
function(event, params) {
    if (this.layer) {
       this.map.raiseLayer(this.layer, this.map.getNumLayers());
       this.map.resetLayersZIndex();
    }
}.bind(this)
```
Instead of referencing this